storage/innobase/handler/ha_innodb.cc

```diff
@@ -2166,6 +2166,11 @@ convert_error_code_to_mysql(
 		return(HA_ERR_RECORD_FILE_FULL);
 
 	case DB_TEMP_FILE_WRITE_FAIL:
+		/* This error can happen during
+		ALTER TABLE..ALGORITHM=COPY or bulk insert operation */
+		innodb_transaction_abort(thd,
+					 innobase_rollback_on_timeout,
+					 error);
```

Comment:
Can it happen during CREATE…SELECT? Would it be accurate to replace `ALTER TABLE..ALGORITHM=COPY` with `copy_data_between_tables()`?